commons-db/src/main/java/com/palantir/nexus/db/sql/SqlConnection.java

```diff
@@ -33,6 +33,7 @@
     boolean selectExists(String key, Object... vs) throws PalantirSqlException, PalantirInterruptedException;
     boolean selectExists(final RegisteredSQLString sql, Object... vs) throws PalantirSqlException, PalantirInterruptedException;
     int selectIntegerUnregisteredQuery(String sql, Object... vs) throws PalantirSqlException, PalantirInterruptedException;
+    long selectLongUnregisteredQuery(String sql, Object... vs) throws PalantirSqlException, PalantirInterruptedException;
```

Comment:
Do we have impl for this? Are there other places we should be reading as longs that are time bombs?